tests/utils/exec/rosacli/ocm_resource_service.go

```diff
@@ -55,6 +55,7 @@ type OCMResourceService interface {
 
 	ListOIDCConfig() (OIDCConfigList, bytes.Buffer, error)
 	ListInstanceTypes() (InstanceTypesList, bytes.Buffer, error)
+	ListClusters(flags ...string) (ClusterList, bytes.Buffer, error)
```

Comment:
Why not reusing existing/extending method in Cluster service ?
https://github.com/openshift/rosa/blob/master/tests/utils/exec/rosacli/cluster_service.go#L21

Reply:
My bad, I wasn't aware about this
i have pushed the fix, could you please take a look? Thanks!